how2meet/ui/frontend.py

```diff
@@ -1,40 +1,124 @@
+    with ui.header().classes('justify-between text-white'):
+        ui.label('How2Meet').classes('font-bold')
+        ui.label(navtitle)
+        with ui.row():
+            header_links = {
+                'Home': '/',
+                'Events': '/events',
+                'Settings': '/settings',
+            }
+            for link_text, link_target in header_links.items():
+                if navtitle != link_text:
+                    ui.link(link_text, link_target).classes(replace='text-white')
 
-@ui.page("/show")
-def show():
-    ui.label("Hello, FastAPI!")
+    with ui.footer().classes('justify-start text-white'):
+        ui.link('Donate', 'https://www.buymeacoffee.com/').classes('text-white')
+        ui.link('GitHub', 'https://www.github.com/BenMemberg/how2meet').classes('text-white')
+    yield
 
-    # NOTE dark mode will be persistent for each user across tabs and server restarts
-    ui.dark_mode().bind_value(app.storage.user, "dark_mode")
-    ui.checkbox("dark mode").bind_value(app.storage.user, "dark_mode")
+@ui.page("/settings")
+def settings():
+    """Settings page"""
+    with frame('Settings'):
+        with ui.column().classes("absolute-center"):
+            # NOTE dark mode will be persistent for each user across tabs and server restarts
+            ui.dark_mode().bind_value(app.storage.user, "dark_mode")
+            ui.checkbox("dark mode").bind_value(app.storage.user, "dark_mode")
 
+@ui.page("/new_event/{event_id}")
+def new_event(event_id: uuid.UUID):
+    """New event creation page
+
+    Args:
+        event_id (uuid.UUID): The unique ID of the event to be created.
 
-@ui.page("/events_list/")
-def show_events():
-    # Use `next() as workaround AttributeError: 'generator' object has no attribute 'query'
-    # (FastAPI makes initial call to generator object to get Session object using `Depends()`
+    Returns:
+        None
+    """
+    def add_guest():
+        with ui.row().classes("w-full"):
+            ui.input("Name").classes("w-1/3")
+            ui.input("Email").classes("w-1/3")
+            ui.input("Phone Number").classes("w-1/3")
+
+    with frame('New Event'):
+        with ui.column().classes("w-full"):
+            with ui.row().classes("w-1/2"):
+                ui.input("Event Name").classes("w-full")
+            with ui.row():
+                ui.input("Password", password=True, password_toggle_button=True)
+            with ui.row():
+                with ui.expansion("Expand"):
+                    ui.date()
+                    ui.time()
+            with ui.row():
+                ui.input("Description")
+            with ui.row().classes("w-half"):
+                with ui.column():
+                    ui.button("Add guest", on_click=add_guest)
+
+        ui.button("Save", on_click=lambda: ui.notify("Event details would be written to DB here"))
+        ui.button("Back", on_click=lambda: ui.open("/"))
+
+@ui.page("/events")
```

Comment:
fyi this is what it looks like when you don't have a working db lol. What does it look like for you?

<img width="769" alt="Screenshot 2024-01-03 at 10 47 55 PM" src="https://github.com/BenMemberg/how2meet/assets/19996600/48d3ea39-6ecc-4406-964b-7264bfd5bd94">

Reply:
That's what it looks like to me. I didn't make the db work for this PR. Out of scope